### PR TITLE
Some changes in text + images

### PR DESCRIPTION
**Pull request by LiraLemur**
Added more steps, changed texts a little and added images (require changing the link if merged).

Link to the scenario: https://katacoda.com/sarag/scenarios/iroha-transfer-asset

### CODE REVIEW

File: iroha-transfer-asset/intro.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-This is a Linux-based sandbox that runs an instance of Hyperledger Iroha v1.0 and demonstrates how users can interact with it's API using Python library.
+This is a Linux-based sandbox that runs Hyperledger Iroha v1.0.
```
Comment (bakhtin) [inline iroha-transfer-asset/intro.md]: I suggest to mention that this installation of iroha consists of a single node.

File: iroha-transfer-asset/step1.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+So both of them can perform any of the available actions in the system. 
+
+There is also an asset -- `coin#test`:
+
+![Coin](assets/coin.png)
+
+Alice, Bob and the asset are parts of the same domain - `test`. 
+There might be other ones in the system, representing different households, companies or institutions.
+
+There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so Iroha could process commands written within .py file.
```
Comment (bakhtin) [inline iroha-transfer-asset/step1.md]: Iroha itself does not process commands written in Python. We interact with Iroha using Python library.

File: iroha-transfer-asset/step3.md [hunk 3]
```diff
@@ -1,31 +1,28 @@
-Let's create another file:
+Now as Alice has her account and a private key, she can send commands to the system.
```
Comment (bakhtin) [inline iroha-transfer-asset/step3.md]: This is a bit misleading. People would think that the previous script creates Alice's account. In fact, we just configure a library to use that particular private key of Alice that was pre-configured inside a Genesis block. Shall we mention this somewhere?

File: iroha-transfer-asset/step3.md [hunk 4]
```diff
@@ -1,31 +1,28 @@
-Let's create another file:
+Now as Alice has her account and a private key, she can send commands to the system. 
 
-`touch transfer-assets.py`{{execute}}
+This next snippet defines a list of commands that will be wrapped into a transaction and sent to Iroha. 
+With `AddAssetQuantity` command Alice will issue a certain amount of the asset `coin#test`. 
 
-And open it:
+You can wrap other commands into transactions - full list of commands and queries can be found in [Iroha docs](https://iroha.readthedocs.io/en/latest/api/index.html). 
+There you can also find parameters that need to be tranferred along with certain commands.
```
Comment (bakhtin) [inline iroha-transfer-asset/step3.md]:
```suggestion
There you can also find parameters that need to be transferred along with certain commands.
```

File: iroha-transfer-asset/step2.md [hunk 5]
```diff
@@ -1,4 +1,10 @@
-The code snippet below creates a connection object, configures private key of user Alice and sets up helper methods to allow tracing of each stage the transaction passes in Iroha. The snippet can be directly copied to a text editor right next to it. It is automatically saved and can be modified if you feel it.
+The code snippet below:
+* creates a connection object
+* configures private key of user Alice, so Alice could confirm transactions
```
Comment (bakhtin) [inline iroha-transfer-asset/step2.md]:
```suggestion
* configures private key of user Alice, so Alice could sign transactions
```

File: iroha-transfer-asset/step6.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+Let's make sure that Bob received our transfer by sending a query about his account balance.
+
+As usual, by creating another file:
+
+`touch get-account-assets.py`{{execute}}
+
+And open it:
```
Comment (bakhtin) [inline iroha-transfer-asset/step6.md]:
```suggestion
And opening it:
```

File: iroha-transfer-asset/step7.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+**That way, during this scenario we have:**
+	* created key for Alice's account
```
Comment (bakhtin) [inline iroha-transfer-asset/step7.md]: No, we don't. As I mentioned previously, it was created in Genesis block

File: iroha-transfer-asset/step7.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+	* issued coins
+	* sent them to Bob's account
+	* made sure they reached it
+
+Now you can check out the rest of the instructions for getting strated with Iroha in our [documentation](https://iroha.readthedocs.io/en/latest/getting_started/index.html) and if you have any questions, you can also check out other resources or contact us in chats:
```
Comment (bakhtin) [inline iroha-transfer-asset/step7.md]:
```suggestion
Now you can check out the rest of the instructions for getting started with Iroha in our [documentation](https://iroha.readthedocs.io/en/latest/getting_started/index.html) and if you have any questions, you can also check out other resources or contact us in chats:
```

**Review by bakhtin — CHANGES_REQUESTED** [review]

File: iroha-transfer-asset/intro.md (hunk 1 above)
Comment (LiraLemur) [reply iroha-transfer-asset/intro.md]: fixed

File: iroha-transfer-asset/step3.md (hunk 3 above)
Comment (LiraLemur) [reply iroha-transfer-asset/step3.md]: Fixed (hopefully)

**Comment by LiraLemur** [comment]
Hopefully, I've fixed it all. 
Please check

File: iroha-transfer-asset/step1.md [hunk 9]
```diff
@@ -1,11 +1,24 @@
...
+So both of them can perform any of the available actions in the system. 
+
+There is also an asset -- `coin#test`:
+
+![Coin](assets/coin.png)
+
+Alice, Bob and the asset are parts of the same domain - `test`. 
+There might be other ones in the system, representing different households, companies or institutions.
+
+There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so commands written within .py file could be processed.
```
Comment (bakhtin) [inline iroha-transfer-asset/step1.md]: It is still a bit confusing. It is a Python library that helps us to interact with Iroha. `.py` files can be processed without the library itself. Can we rephrase to something like:
```suggestion
There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so we can interact with Iroha from within a Python code
```

**Review by bakhtin — APPROVED** [review]

**Comment by LiraLemur** [comment]
The suggestion looked great.

**Comment by bakhtin** [comment]
Cool. Merging it now

**Pull request merged**